tests/Jellyfin.Model.Tests/Extensions/StringHelperTests.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using MediaBrowser.Model.Extensions;
+using Xunit;
+
+namespace Jellyfin.Model.Tests.Extensions
+{
+    public class StringHelperTests
+    {
+        [Theory]
+        [InlineData("", "")]
+        [InlineData("banana", "Banana")]
+        [InlineData("Banana", "Banana")]
+        [InlineData("ä", "Ä")]
+        public void FirstToUpperTest(string str, string result)
```

Comment:
This should follow the same naming scheme, perhaps as below. I also think these parameter names could be a bit more descriptive

```suggestion
        public void FirstToUpper_ReturnsCorrectValue(string input, string expectedResult)
```